app/jobs/tweet_stat_adjustment_job.rb

```diff
@@ -0,0 +1,8 @@
+class TweetStatAdjustmentJob < ApplicationJob
+  queue_as :tweet_stat_adjustment
```

Comment:
`tweet_refresh_stats` perhaps?

Reply:
`tweet_tag_refresh_stats` I believe is OK also